client/my-sites/domains/domain-management/list/style.scss

```diff
@@ -134,10 +134,16 @@
 		}
 	}
 }
-// Custom layout width for the bulk domains pages
-.is-bulk-domains-page .layout__content {
+
+// The /domains/manage/:site page
+.is-bulk-site-domains-page .layout__content {
 	overflow: auto;
```

Comment:
this is neccessary for now, and needs to be fixed as part of 7114-gh-Automattic/dotcom-forge#issuecomment-2111416130